lib/timers.js

```diff
@@ -409,6 +393,26 @@ function processImmediate() {
   }
 }
 
+function tryOnImmediate(immediate, queue) {
```

Comment:
I've been using the `try` prefix with the `try-catch/finally` isolation PRs I've submitted in the past as that seems (to me) to make it more clear as to what the isolation functions are doing. \*shrug\*